test/behaviour/resolution/reasoner/concept_inequality/BUILD

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2020 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+package(default_visibility = ["//test/behaviour:__subpackages__"])
+load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+
+java_test(
+    name = "test",
+    srcs = [
+        "ConceptInequalityTest.java",
+    ],
+    test_class = "grakn.core.test.behaviour.resolution.reasoner.concept_inequality.ConceptInequalityTest",
```

Comment:
can we just call the package `comparison` or something? two word names not so neat :P

Reply:
I followed the convention from Graql language tests where we put each test feature into its own package named identically to the feature.